src/cli.coffee

```diff
@@ -15,18 +9,25 @@ options = docopt """
   """
 
 
-generateConfig = ->
-  break for extension in extensions when options[extension]
+{docopt} = require 'docopt'
+options = docopt usage
+
+
+generateConfig = (extension) ->
+  asyncHandlers = require 'async-handlers'
```

Comment:
`asyncHandlers` is used in the callback, not here. Thinking your approach all the way to the end, shouldn't it be required on line 24?